sklearn/ensemble/_hist_gradient_boosting/tests/test_warm_start.py

```diff
@@ -115,11 +116,12 @@ def test_warm_start_early_stopping(GradientBoosting, X, y):
 ])
 def test_warm_start_equal_n_estimators(GradientBoosting, X, y):
     # Test if warm start with equal n_estimators does nothing
-    gb_1 = GradientBoosting(max_depth=2)
+    gb_1 = GradientBoosting(max_depth=2, n_iter_no_change=5)
```

Comment:
Did you mean `early_stopping=False`? I don't think the current change has any effect.

Reply:
Yes!